manifest.json

```diff
@@ -16,8 +16,7 @@
 		"default_title": "CGPA"
 	},
 	"background": {
-		"scripts": ["browser-polyfill.js", "background.js"],
-		"persistent": false
+		"service_worker": "worker-wrapper.js"

```

Comment:
Firefox MV3 is currently in beta and does not support background service worker.
Lets wait for merging this. We can merge this when firefox supports this.
If that does not happen before chrome ends mv2 support, we can create a script to generate 2 different manifest files.